src/scripts.js

```diff
@@ -346,188 +232,302 @@ function makeSleepHTML(id, sleepInfo, userStorage, method) {
+    const form = document.querySelector(`${className}`);
+    loadMonitorData();
+    displaySuccessMessage(form);
+    form.reset();
+    setTimeout(() => { form.lastChild.innerHTML = '' }, 5000);
+  }
+
+  function displaySuccessMessage(form) {
+    const successMessage = `<p>You have successfully posted your data!</p>`;
+    form.insertAdjacentHTML('beforeend', successMessage);
+  }
+
+  function evaluateSleepInput() {
+    const dateInput = document.querySelector('.sleep-date-input');
+    const sleepAmount = document.querySelector('.hours-slept-input');
+    const sleepQuality = document.querySelector('.sleep-quality-input');
+    if (dateInput.value !== "" && sleepAmount.value !== "" && sleepQuality.value !== "") {
+      const dataToPost = createSleepObject(dateInput, sleepAmount, sleepQuality);
+      postSleepSubmission(dataToPost);
+    }
+  }
+
+  function createSleepObject(dateInput, sleepAmount, sleepQuality) {
+    const formattedDate = dateInput.value.replace(/-/g,"/");
+    return {"userID": user.id, "date": formattedDate, "hoursSlept": parseInt(sleepAmount.value), "sleepQuality": parseInt(sleepQuality.value)}
+  }
+
+  function postSleepSubmission(dataToPost) {
+    fetch("https://fe-apps.herokuapp.com/api/v1/fitlit/1908/sleep/sleepData", {
+      method: 'POST',
+      headers: {
+  	'Content-Type': 'application/json'
+    },
+      body: JSON.stringify(dataToPost),
+    })
+    .then(response => response.json())
+    .then(message => handlePostSuccess('.sleep'))
+    .catch(error => console.log(error.message))
+  }
+
+  function evaluateActivityInput() {
+    const dateInput = document.querySelector('.activity-date-input');
+    const stepCount = document.querySelector('.steps-walked-input');
+    const minutesActive = document.querySelector('.minutes-active-input');
+    const stairCount = document.querySelector('.stairs-input');
+    if (dateInput.value !== "" && minutesActive.value !== "" && stepCount.value !== "" && stairCount.value !== "") {
+      const dataToPost = createActivityObject(dateInput, stepCount, minutesActive, stairCount);
+      postActvitySubmission(dataToPost);
+    }
+  }
+
+  function createActivityObject(dateInput, stepCount, minutesActive, stairCount) {
+    const formattedDate = dateInput.value.replace(/-/g,"/");
+    return {"userID": user.id, "date": formattedDate, "numSteps": parseInt(stepCount.value), "minutesActive": parseInt(minutesActive.value), "flightsOfStairs": parseInt(stairCount.value)}
+  }
+
+  function postActvitySubmission(dataToPost) {
+    console.log(dataToPost);
+    fetch("https://fe-apps.herokuapp.com/api/v1/fitlit/1908/activity/activityData", {
+      method: 'POST',
+      headers: {
+  	'Content-Type': 'application/json'
+    },
+      body: JSON.stringify(dataToPost),
+    })
+    .then(response => response.json())
+    .then(message => handlePostSuccess('.activity'))
+    .catch(error => console.log(error.message))
+  }
+
+//not being used!
```

Comment:
I saw this too.  I think we need to have a conversation about what the purpose of this function was supposed to be before entirely removing it, but nice catch!